packages/cli/core/src/index.ts

```diff
@@ -26,6 +26,7 @@ import {
   loadUserConfig,
   UserConfig,
   ToolsConfig,
+  DeployConfig,
```

Comment:
If DeployConfig is added, ,maybe you could also export other Config at one time.
```javascript
export * from './config';
```

Reply:
the original export utilizes Typescript `export type { ... }` to ensure all related code is removed at runtime.
see [Type Only Imports and Export](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export) for more details.

I've moved the maintenance of 'export type' to core/config/index.ts. This will allow us to use `export * from './config'` while making sure all types are removed at runtime. This will also be better for maintenance since contributors will be able to see the `export` usage in the same file and won't need to modify other files

Reply:
Nice work out there:)